.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Users Mailing List
```

Comment:
I'd un-title-case this ("Users mailing list" and "Prometheus community" below) so it's consistent in casing with the first two entries ("Bug report" and "Feature request").